### PR TITLE
Navigation refactoring

### PR DESCRIPTION
**Pull request by thomasio101**

### CODE REVIEW

**Comment by thomasio101** [comment]
[Navigatie-UI verbeteren](https://trello.com/c/7x2O3HJU/34-navigatie-ui-verbeteren)

File: WebdevPeriod3/wwwroot/js/site.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
-﻿document.addEventListener("DOMContentLoaded", function (event) {
-    document.getElementById("nav-toggle").addEventListener("click", (e) => {
-        document.getElementById("side-nav").classList.toggle("hidden");
+﻿$(() => {
```
Comment (stefvanhouten) [inline WebdevPeriod3/wwwroot/js/site.js:1]: Is there a reason we re-added jQuery into the project besides toggling the class of an element?
If it is only used here and its only purpose is toggling an element I would prefer just using vanilla JS.

**Comment by stefvanhouten** [comment]
It also seems like jQuery isn't properly working.
afbeelding

It seems like the wwwroot/lib is missing from the project. Are we supposed to download a version of jQuery ourselves? It shouldn't be an issue to put the minified version of jquery on github though.

**Comment by thomasio101** [comment]
@stefvanhouten this is a bug in VS2019. You need to open `libman.json` and save (`ctrl + s`). This will trigger the installation process.

**Pull request merged**